packages/discord.js/src/managers/GuildScheduledEventManager.js

```diff
@@ -99,6 +102,7 @@ class GuildScheduledEventManager extends CachedManager {
         description,
         entity_type: entityType,
         entity_metadata,
+        image: image ? await DataResolver.resolveImage(image) : undefined,
```

Comment:
```suggestion
        image: image && await DataResolver.resolveImage(image),
```

Reply:
ah yes setting image to null to remove the image on edit, that slipped